src/styles/header.scss

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  background: $background-header;
+  box-shadow: $box-shadow;
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    display: flex;
+    margin-right: 20px;
+    font-weight: 500;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.nav__link {
+  align-items: center;
+  justify-content: center;
+  padding: 23px 0;
+  font-size: 12px;
+  color: $default-color-link;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  &:hover {
+    color: $color-active-link;
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $color-active-link;
+
+  &::after {
+    position: absolute;
+    height: 4px;
+    width: 100%;
+    flex-shrink: 0;
+    left: 0;
+    right: 0;
```

Comment:
The use of both 'left: 0;' and 'right: 0;' together is redundant when setting the width to 100%. You can remove one of them to simplify the code.